### PR TITLE
Update constaint module dependency and bump minor version

### PR DESCRIPTION
**Pull request by daneshk**
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec
- [ ] Checked native-image compatibility

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,10 +1,10 @@
 org.gradle.caching=true
 group=io.ballerina.stdlib
-version=3.7.2-SNAPSHOT
+version=3.8.0-SNAPSHOT
 ballerinaLangVersion=2201.8.0
 stdlibTaskVersion=2.5.0
 stdlibTimeVersion=2.4.0
-stdlibConstraintVersion=1.4.0
+stdlibConstraintVersion=1.5.0
```
Comment (TharmiganK) [inline gradle.properties:7]: This Constraint version is compatible with Ballerina `2201.8.0`, this new minor version has some new features.
@ThisaruGuruge In that case do we need to bump to a new minor version?

Comment (daneshk) [reply gradle.properties:7]: yes, we are bumping to a new minor version and going to release with update 08

Comment (TharmiganK) [reply gradle.properties:7]: Any particular reason to do that?

Comment (TharmiganK) [reply gradle.properties:7]: Had a discussion offline and agreed to bump a minor version for update 9

**Review by TharmiganK — APPROVED** [review]

**Pull request merged**